networking/networking.go

```diff
@@ -229,18 +233,41 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
-	if len(n.nets) == 0 {
-		return nil
+// GetDefaultNet iterates through all loaded networks and returns either
+// the first network that has masquerading enabled,
+// or the last network in case there is no masqueraded one,
+// or an error if no network was loaded.
```

Comment:
Yes, that would be easier to find the explanation when reading the code. So readers would not have to use `git annotate`.